plugins/automation/admin/assets/javascripts/admin/templates/admin-plugins-discourse-automation-new.hbs

```diff
@@ -1,52 +1,33 @@
 <section class="discourse-automation-form new">
-  <form class="form-horizontal">
-    <FormError @error={{error}} />
 
-    <div class="control-group">
-      <label class="control-label">
-        {{i18n "discourse_automation.models.automation.name.label"}}
-      </label>
+  <div
+    class="admin-section-landing__header"
+    {{did-insert this.resetFilterText}}
+  >
+    <h2>{{i18n "discourse_automation.select_script"}}</h2>
 
-      <div class="controls">
-        <TextField
-          @value={{form.name}}
-          @type="text"
-          @autofocus="autofocus"
-          @name="automation-name"
-          class="input-large"
-        />
-      </div>
-    </div>
-
-    <div class="control-group">
-      <label class="control-label">
-        {{i18n "discourse_automation.models.automation.script.label"}}
-      </label>
+    <input
+      type="text"
+      placeholder={{i18n "discourse_automation.filter_placeholder"}}
+      {{on "input" this.updateFilterText}}
+      class="admin-section-landing__header-filter"
+    />
+  </div>
 
-      <div class="controls">
-        <DropdownSelectBox
-          @value={{form.script}}
-          @content={{model.scriptables.content}}
-          @onChange={{fn (mut form.script)}}
-          @options={{hash
-            showCaret=true
-            filterable=true
-            none="discourse_automation.select_script"
-          }}
-          class="scriptables"
-        />
-      </div>
+  {{#if this.redirected}}
```

Comment:
This is a bit odd, in that if you have no automations and you refresh the page, you don't see the info alert. Minor thing, but maybe we can check the model length here?